vm/src/types/slot.rs

```diff
@@ -150,6 +150,8 @@ pub(crate) type DescrSetFunc =
 pub(crate) type NewFunc = fn(PyTypeRef, FuncArgs, &VirtualMachine) -> PyResult;
 pub(crate) type DelFunc = fn(&PyObject, &VirtualMachine) -> PyResult<()>;
 
+pub use crate::builtins::object::{generic_getattr, generic_setattr};
```

Comment:
Oh that was for part 2 of this, for filling in the default get/setattro slot. Is it ok if I leave it? I think it's more hassle to take it out and I think in general it'd be good to have definitive *function pointers* for generic slots rather than a bunch of wrappers that call those functions. Like 20 `impl Getattro for _ { fn getattro() { vm.generic_getattro() } }` vs just directly using a `generic_getattr` function.